caproto/threading/client.py

```diff
@@ -440,9 +443,13 @@ def send(self, port, *commands):
             else:
                 specified_port = port
             try:
+                tags = {'role': 'CLIENT',
+                        'their_address': (host, specified_port),
+                        'our_address': self.udp_sock.getsockname()[:2],
```

Comment:
Do we need to  compute this every time? Can we access it on `self.broadcaster.our_address`?

Reply:
yes